start.ps1

```diff
@@ -75,8 +75,8 @@ function install {
 
 function set_env {
     # set all env vars in .env file
-    if (-not (Test-Path "$($repo_root)\.env")) {
-        Write-Host "!!! ", "Setting environment variables from .env!" -ForegroundColor red
+    if ((Test-Path "$($repo_root)\.env")) {
+        Write-Host "!!! ", ".env file must be prepared!" -ForegroundColor red
```

Comment:
i'd revert that line back to "Setting environment variables from .env!" since we changed from throwing an error if `.env` doesn't exist to set the env if it is present